css-lean-string.json

```diff
@@ -45,14 +47,25 @@
               "name": "meta.selector.css",
               "patterns": [{ "include": "source.css#selector" }]
             },
-            "2": { "name": "punctuation.section.property-list.begin.bracket.curly.css" }
+            "2": {
+              "name": "punctuation.section.property-list.begin.bracket.curly.css"
+            }
           }
         },
         {
           "match": "((?:\\$[^\\{])?[^;\\$\\s`:'\"\\}]+)\\s*(:)?\\s*(['\"][^'\"]*['\"]|(?:\\$[^\\{])?[^;\\$\\}`]+)?",
           "name": "meta.property-list.css",
           "captures": {
-            "1": { "name": "support.type.property-name.css" },
+            "1": {
+              "name": "meta.property-name.css",
```

Comment:
does it not work if we keep `support.type.property-name.css` here rather than using `.*` as the second match?

Reply:
I thought you wanted both?

Reply:
true, I'll test it out and decide whether I still want it